cablelabs/lpwanserver
# show server message in edit form error banner  when MW not authorized

#### What does this PR do?
show server message in edit form error banner  when NW not authorized
remove unneeded npm dependencies

#### Do you have any concerns with this PR?
no

#### How can the reviewer verify this PR?
create NW with bad auth info, edit it, you should see server auth message in error banner

#### Any background context you want to provide?
no

#### Screenshots or logs (if appropriate)

![image](https://user-images.githubusercontent.com/5873306/44311413-44db8000-a3a4-11e8-9adb-7e487282caad.png)

#### Questions:
- Have you connected this PR to the issue it resolves?
#42 

- Does the documentation need an update?
no

- Does this add new dependencies?
no

- Have you added unit or functional tests for this PR?
no
